### PR TITLE
Documentation & Testing

### PR DESCRIPTION
**Pull request by yechenh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,314 @@
...
+token-level edit operations. The goal of this project is to improve the 
+performance of this model specifically for the short-sentence-and-phrase 
+summarization task. 
+
+Our improvement of the model includes adding part-of-speech (POS) tags to 
+BERT embeddings (which involves pre-training BERT with POS tags), 
+customizing loss function and loss weights, applying masks, and 
+hyperparameter tuning. 
+
+To address the lack of grammar evaluation in existing performance metrics
```
Comment (XiangZeng2020) [inline README.md:27]: Add a link to the checker file.

File: README.md [hunk 2]
```diff
@@ -1 +1,314 @@
...
+there are several prerequisites:
+- Create or have access to a Google Storage Bucket. Currently, the GCP bucket 
+path is set to gs://trained_models_yechen/. If you create another bucket, change 
+the path by changing the GCP_BUCKET variable in streamline_training.py.
+-  Set up a virtual machine to run the script on. Follow this [guide](
+https://docs.google.com/document/d/1oV8Swp_BDfmDHkhSkWb2wo_ZhC9jIP-Lk7kCbYvdYTM/edit#heading=h.o18hkt51hrci) 
+to set up a virtual machine on GCP.
+- If you plan to train with a Cloud TPU, follow this [guide](
+https://docs.google.com/document/d/1PlCB6DOH8LUBsN8UcgPxzds9MqPRFIPs_rht9fWqbAA/edit?usp=sharing) 
+to set up a TPU on GCP.
```
Comment (XiangZeng2020) [inline README.md:112]: Add a note that the TPU name and compute engine name must be the same.
Also add a note that if the zones for TPU and compute engine are different, need to set --tpu_zone with the TPU zone.

File: README.md (hunk 1 above)
Comment (yechenh) [reply README.md:27]: Done

File: README.md (hunk 2 above)
Comment (yechenh) [reply README.md]: Added both notes to the guide. Added the first note to the readme file.

**Pull request merged**